### ISSUE TITLE
RLE example does not work - 'bitWidth' does not exist in type 'FieldDefinition'

### ISSUE THREAD

**Issue by benjamin-brady** [LibertyDSNP/parquetjs#106]
### Steps to reproduce

- Install parquetjs from npm
- Try to compile the following code from the README:

```typescript    
import parquetjs from "@dsnp/parquetjs"
var schema1 = new parquetjs.ParquetSchema({
age: { type: 'UINT_32', encoding: 'RLE', bitWidth: 7 },
});
```

### Expected behaviour
It should at least compile.

### Actual behaviour
Error because there is no bitWidth property on the FieldDefinition type.
```
Object literal may only specify known properties, and 'bitWidth' does not exist in type 'FieldDefinition'.ts(2353)
declare.d.ts(14, 5): 
The expected type comes from this index signature.
(property) bitWidth: number
```

**Comment by wilwade** [comment]
@benjamin-brady Looks like there was an upstream doc issue.

tldr: Use `typeLength` instead of `bitWidth`

https://github.com/ironSource/parquetjs/issues/100

I'll put up a PR to fix it with a simple test as well.
